Firestore/core/src/remote/remote_objc_bridge.cc

```diff
@@ -278,7 +278,7 @@ DatastoreSerializer::EncodeAggregateQueryRequest(
     const std::vector<AggregateField>& aggregates,
     absl::flat_hash_map<std::string, std::string>& aliasMap) const {
   Message<google_firestore_v1_RunAggregationQueryRequest> result;
-  auto encodedTarget = serializer_.EncodeQueryTarget(query.ToTarget());
+  auto encodedTarget = serializer_.EncodeQueryTarget(query.ToAggregateTarget());
```

Comment:
oh my! I would have missed this completely. Are there any other instances of `ToTarget()` that need to be updated?

Reply:
cool,thanks